### PR TITLE
fix(lotdetails): L3-3327 minor changes to input and accordion to match compendium

### PR DESCRIPTION
**Pull request by sgrund14**
**Jira ticket**

[L3-3327](https://phillipsauctions.atlassian.net/browse/L3-3327)

**Screenshots**
**Before**
Screenshot 2024-10-28 at 10 30 36 AM
**After**
Screenshot 2024-10-28 at 10 31 56 AM

**Summary**

Match invalid email styles to compendium

**Change List (describe the changes made to the files)**

- keep invalid text black, increase font weight of input text, red outline on focused invalid input
- add onOpen/onClose props to accordion item

**Acceptance Test (how to verify the PR)**

- Add step by step instructions on how to verify the change

**Regression Test**

- (Optional) Add verification steps to make sure this PR doesn't break old functionality

**Evidence of testing**

- Post logs, screenshots, etc

**Things to look for during review**

- [ ] PR title should correctly describe the most significant type of commit. I.e. `feat(scope): ...` if a `minor` release should be triggered.
- [ ] All commit messages follow convention and are appropriate for the changes
- [ ] All references to `phillips` class prefix are using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] Document all props with jsdoc comments
- [ ] All strings should be translatable.
- [ ] Unit tests should be written and should have a coverage of 90% or higher in all areas.

New Components

- [ ] Are there any [accessibility considerations](https://www.w3.org/WAI/ARIA/apg/patterns/) that need to be taken into account and tested?
- [ ] Default story called "Playground" should be created for all new components
- [ ] Create a jsdoc comment that has an Overview section and a link to the Figma design for the component
- [ ] Export the component and its typescript type from the `index.ts` file
- [ ] Import the component scss file into the `componentStyles.scss` file.

[L3-3327]: https://phillipsauctions.atlassian.net/browse/L3-3327?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Input/_input.scss [hunk 1]
```diff
@@ -15,8 +15,10 @@ $lg: #{$px}-input--lg;
 
   &__label {
     color: $pure-black;
+    display: flex;
     font-size: 0.75rem;
     font-variation-settings: 'wght' 600;
+    gap: 0.25rem;
```
Comment (scottdickerson) [inline src/components/Input/_input.scss:21]: I see a gap of spacer-xsm in Figma in the parent div

File: src/components/Input/_input.scss [hunk 2]
```diff
@@ -30,9 +32,14 @@ $lg: #{$px}-input--lg;
     border: 1px solid $keyline-gray;
     border-radius: 0.1875rem;
     font-size: 0.8125rem;
+    font-variation-settings: 'wght' 600;
     margin-bottom: 0.25rem;
     padding: $padding-xsm;
```
Comment (scottdickerson) [inline src/components/Input/_input.scss:37]: seems like the bottom padding to the Error is padding-micro in the Figma?

Comment (sgrund14) [reply src/components/Input/_input.scss:37]: sure, I can add in

this would stay the same though, since it's not the error?

Comment (sgrund14) [reply src/components/Input/_input.scss:35]: also this padding value is for the inner padding of the input

**Review by davidicus — APPROVED** [review]
LGTM. The padding thing is minor IMO and does not really align to a defined token. I have left a comment on the Figma to see if the difference is intentional.

File: src/components/Input/_input.scss (hunk 2 above)
Comment (sgrund14) [reply src/components/Input/_input.scss:35]: Screenshot 2024-10-30 at 11 00 20 AM

🤔

Comment (sgrund14) [reply src/components/Input/_input.scss:35]: maybe some of that space is coming from the line height of the input

**Comment by sgrund14** [comment]
hm, publish job failing...

**Pull request merged**